pkg/api/http/actors.go

```diff
@@ -90,7 +98,7 @@ func (a *api) constructActorEndpoints() []endpoints.Endpoint {
 				Name:                 endpoints.EndpointGroupActors,
 				Version:              endpoints.EndpointGroupVersion1,
 				AppendSpanAttributes: appendActorInvocationSpanAttributesFn,
-				MethodName:           actorInvocationMethodNameFn,
+				MethodName:           methodNameFn,
```

Comment:
Should we be using a new field on `EndpointGroup` for this?

Reply:
I don't have a strong opinion here. Having a single attribute that handles the logic of low/high cardinality seems to be ok to me.